src/tests/stats/test_var.f90

```diff
@@ -154,7 +154,7 @@ program test_var
     call assert( all( abs( var(d, 1, d < 11) -&
                  [20._dp / 3._dp, 20._dp / 3._dp, 0.5_dp]) < dptol))
     call assert( all( abs( var(d, 2, d < 11) -&
-                 [19.0_dp, 43._dp / 3._dp, 0.5 _dp, 0.5_dp]) < dptol))
+                 [19.0_dp, 43._dp / 3._dp, 0.5_dp, 0.5_dp]) < dptol))
```

Comment:
How is it that this even compilers with gfortran?

Reply:
I have no idea. I tested different versions of gfortran and it compiled fine with all of them.

Reply:
(I wonder if gfortran makes the space in `0.5 _dp` optional.)